assets/src/helpers.js

```diff
@@ -472,3 +449,16 @@ export const hasMinimumStoryPosterDimensions = ( media ) => {
 		( media.width >= minWidth && media.height >= minHeight )
 	);
 };
+
+/**
+ * Whether the image has the minimum width for a featured image.
+ *
+ * This should have a width of at least 1200 pixels
+ * to satisfy the requirement of Google Search for Schema.org metadata.
+ *
+ * @param {Object} media A media object with width and height values.
+ * @return {boolean} Whether the media has the minimum dimensions.
+ */
+export const hasMinimumFeaturedImageWidth = ( media ) => {
+	return ( media.width && media.width >= 1200 );
```

Comment:
The 1200 value is being used for the width, but there are more requirements than this we should also account for. [Namely](https://developers.google.com/search/docs/data-types/article#article_types):

> * Images should be at least 1200 pixels wide. ✅ 
> * For best results, provide multiple high-resolution images (minimum of 800,000 pixels when multiplying width and height) with the following aspect ratios: 16x9, 4x3, and 1x1.
> * Images must be in .jpg, .png, or .gif format.

So we should also address these later.

----

I see that `hasMinimumStoryPosterDimensions` is actually accounting for more of [its requirements](https://www.ampproject.org/docs/reference/components/amp-story#new-metadata-requirements), so that is good:

> Each poster image should meet the recommended minimium size:
> * Portrait: 696px x 928px
> * Landscape: 928px x 696px
> * Square: 928px x 928px

Nevertheless, it is good that we are requiring 1200 for the featured image since then it can be re-used for both the poster _and_ the Schema.org metadata. The cropping I believe is handled elsewhere.

Reply:
We should also update the Schema.org markup to output multiple images in the desired aspect ratios:

```json
{
  "@context": "https://schema.org",
  "@type": "NewsArticle",
  "image": [
    "https://example.com/photos/1x1/photo.jpg",
    "https://example.com/photos/4x3/photo.jpg",
    "https://example.com/photos/16x9/photo.jpg"
  ]
}
```

Reply:
> Nevertheless, it is good that we are requiring 1200 for the featured image since then it can be re-used for both the poster and the Schema.org metadata.

There are two different requirements/checks:

* 696x928 for AMP Stories
* 1200 minimum width for regular posts

Did you mean that for AMP Stories we should also require 1200 minimum width?

Reply:
Oh, yes, actually. Sorry, I rewrote that comment a couple times as I discovered some more things. So since there is a separate function specifically for stories, then the 1200 constraint isn't applying. So yes, we should also enforce AMP stories gave a featured image of at least 1200px in order to satisfy the poster requirements and the metadata requirements.

Reply:
Cool, makes sense. I'll bump the width requirement in `hasMinimumStoryPosterDimensions` then.